include/OpenSpaceToolkit/Astrodynamics/NumericalSolver.hpp

```diff
@@ -47,7 +43,8 @@ class NumericalSolver
 
     typedef std::vector<double> StateVector;  // Container used to hold the state vector
     typedef std::function<void(const StateVector&, StateVector&, const double)>
-        SystemOfEquationsWrapper;  // Function pointer type for returning dynamical equation's pointers
+        SystemOfEquationsWrapper;                // Function pointer type for returning dynamical equation's pointers
+    typedef Pair<StateVector, double> Solution;  // Container used to hold the state vector and time
```

Comment:
Not a fan of "Solution" as the name for this but I can't really think of anything better that doesn't collide with State. 🤷

Reply:
Agreed. I went with solution incase if we want to expand this in the future it would be easy. It's also namespaced under NumericalSolver so I'm more okay with it.